domains/homelab.is-an.app.js

```diff
@@ -0,0 +1,13 @@
+addSubDomain({
+  description: 'Backend routing for all my homelab projects',
+  domain: 'is-an.app',
+  subdomain: '*.homelab',
```

Comment:
Wildcards are only for limited use cases. Maybe you can define all required subdomains as nested instead?

Reply:
That will require a new PR everytime I want to add something
Also adding projects here would mean anybody coming to this repo will have a list of apps I am running 
What else could be usecase for a wildcard?

Reply:
> require a new PR

Yea. Instead of this, you can buy a "normal" domain and do what you want with it. But I'm worried about other users because if one time the malware (or something else) appears on one of "your" subdomains (eg. your server will be hacked) - and my domain will be complained about - all users will suffer.

I hope you understand me.

> What else could be usecase for a wildcard?

A special resolving zone, for example.

Reply:
If you want mass subdomains - maybe, this project is not for you. If you have a pet project or so on - you are welcome!

Reply:
hey all pet projects here. will update the PR

Reply:
removed wildcard.